### DISCUSSION TITLE
How many completions doed a kata need to be approved?

### DISCUSSION THREAD

**Discussion by yur-itch** [codewars/codewars.com#2998] [Q&A] [answered]
I started kata series of BF: https://www.codewars.com/collections/logic-from-scratch
3 of them already have 10 completions, which is probably enough for such a language.
Do you know how many completions does a kata need to be approved and is it the key?

**Comment by Kacarott** [comment, marked as answer]
All 4 kata are already approveable, however for them to be approved someone needs to take the time to review them and decide on a rank, so sometimes it can take a bit of time (especially for an esoteric language like BF). I'll try have a look at them soon.
